cmd/scollector/collectors/iterable.go

```diff
@@ -0,0 +1,70 @@
+package collectors
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"github.com/bosun-monitor/statusio"
+)
+
+func init() {
+	registerInit(func(c *conf.Conf) {
+		collectors = append(collectors, &IntervalCollector{
+			F: func() (opentsdb.MultiDataPoint, error) {
+				return cIterableStat(c.Iterable.StatusBaseAddr)
+			},
+			name:     "cIterableStat",
```

Comment:
For better or worse our convention is underscores and lowercase, for example https://github.com/bosun-monitor/bosun/blob/master/cmd/scollector/collectors/fastly.go#L44 , so should be `c_iterable_status`